Tests/MessageGeneratorTests.swift

```diff
@@ -1,5 +1,255 @@
 import UIKit
 import XCTest
+import CoreLocation
+
+@testable import Daylight
 
 class MessageGeneratorTests: XCTestCase {
+    let initialDate: Date = Date(timeIntervalSince1970: 1513854000)
```

Comment:
can you throw in a comment as to what calendrical date this is?

Reply:
Looks like I accidentally deleted it before pushing. 😅